plugins/scaffolder-backend/src/scaffolder/actions/builtin/filesystem/delete.test.ts

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as os from 'os';
+import mockFs from 'mock-fs';
+import { resolve as resolvePath } from 'path';
+import { createFilesystemDeleteAction } from './delete';
+import { getVoidLogger } from '@backstage/backend-common';
+import { PassThrough } from 'stream';
+import fs from 'fs-extra';
+
+const root = os.platform() === 'win32' ? 'C:\\rootDir' : '/rootDir';
+const workspacePath = resolvePath(root, 'my-workspace');
+
+describe('fs:delete', () => {
+  const action = createFilesystemDeleteAction();
+
+  const mockContext = {
+    input: {
+      files: ['unit-test-a.js', 'unit-test-b.js'],
+    },
+    workspacePath,
+    logger: getVoidLogger(),
+    logStream: new PassThrough(),
+    output: jest.fn(),
+    createTemporaryDirectory: jest.fn(),
+  };
+
+  beforeEach(() => {
+    jest.restoreAllMocks();
+
+    mockFs({
+      [workspacePath]: {
+        'unit-test-a.js': 'hello',
```

Comment:
The description tells: "A list of files or _directories_ that will be deleted" but there is no test that deletes a directory. Would be nice to have one to be sure it works. I think this would work: `'a-folder/unit-test-in-a-folder.js2': 'content'`.

Reply:
good point, I've updated the tests: [`3d24e8a` (#6340)](https://github.com/backstage/backstage/pull/6340/commits/3d24e8aecf92d41a1d4916aaa4f69180e38683bc)